jest.config.js

```diff
@@ -22,16 +22,18 @@ module.exports = {
     '^.+\\.(js|jsx)$': 'babel-jest',
   },
   testPathIgnorePatterns: ['<rootDir>/node_modules/'],
+  collectCoverage: true,
   collectCoverageFrom: [
-    'src/**/*.{ts,tsx,js,jsx}',
     'libs/**/*.{ts,tsx,js,jsx}',
-    '!src/**/*.d.ts',
-    '!src/**/*.spec.ts',
-    '!src/**/*.test.ts',
-    '!libs/**/*.d.ts',
-    '!libs/**/*.spec.ts',
-    '!libs/**/*.test.ts',
   ],
+  coverageThreshold: {
+    global: {
+      statements: 94,
+      branches: 83,
+      functions: 91,
+      lines: 97
+    }
+  },
```

Comment:
> so that when we make new additions to the code, you need to add unit tests to keep it up to that level.

In order to achieve this enforcement, can we raise lines to 100? Looks like just 5 lines left 

<img width="721" height="236" alt="Image" src="https://github.com/user-attachments/assets/862e974d-dd4a-4009-afd7-12861e3a3f0d" />